apps/jobs/utils.py

```diff
@@ -220,3 +221,21 @@ def handle_submission_rerun(submission, updated_status):
                 "submitted_image_uri"
             ]
     return message
+
+
+def notify_on_slack(slack_url, slack_data):
```

Comment:
@Ayukha I just realized that we already had this function written in base/utils.py. Can you please use that -> https://github.com/Cloud-CV/EvalAI/blob/master/apps/base/utils.py#L207